<launch>
	<node name="joy_node" pkg="joy" type="joy_node">
		<param name="autorepeat_rate" value="10"/>
	</node>
	<node name="mover" pkg="valkyrie_twist_teleop" type="mover.py" output="screen">
		<param name="initial_leg_aperture" value="0.3" />
		<param name="linear_speed_limit" value="0.01" /> <!--0.1-->
		<param name="angular_speed_limit" value="0.045" /> <!--0.09-->
		<param name="swing_time" value="3.0" /> <!--1.0-->
		<param name="transfer_time" value="1.0" />
	</node>
	<node name="joy_teleop" pkg="valkyrie_twist_teleop" type="joy_teleop.py">
		<remap from="cmd_vel" to="/ihmc_ros/valkyrie/control/cmd_vel" />
	</node>
</launch>
